server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -300,9 +304,9 @@ Issues related to authorization may exist just for some other versions of {jdgse
 ```
 
 +
-. Copy the server to the second location, which will be referred to later as `JDG2_HOME`.
+. Copy the server to the second location, which will be referred to later as `SERVER2_HOME`.
 
-. In the `JDG2_HOME/standalone/configuration/clustered.xml` exchange `site1` with `site2` and vice versa, both in the configuration of `relay` in the JGroups subsystem and in configuration of `backups` in the cache-subsystem. For example:
+. In the `SERVER2_HOME/standalone/configuration/clustered.xml` exchange `site1` with `site2` and vice versa, both in the configuration of `relay` in the JGroups subsystem and in configuration of `backups` in the cache-subsystem. For example:
```

Comment:
`SERVER2_HOME/standalone/configuration/infinispan.xml` instead of "clustered.xml"
No more subsystem
The relay element looks like this (line 313-316)

```
<stack name="xsite" extends="udp">
     <relay.RELAY2 site="site2" xmlns="urn:org:jgroups"/>
     <remote-sites default-stack="global-cluster">
        <remote-site name="site1"/>
        <remote-site name="site2"/>
     </remote-sites>
  </stack>
```